recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -152,8 +155,3 @@ INSANE_SKIP_${PN}-dbg += "dev-so"
 # ----------------------------------------------------------------------------
 
 RDEPENDS_${PN}_rpi = "userland"
-
-# Avoid settings ADNEEDED in LDFLAGS as this can cause the libcompositor.so to drop linking to libEGL/libGLES
-# which might not be needed at first glance but will cause problems higher up in the change, there for lets drop -Wl,--as-needed
-# some distros, like POKY (morty) enable --as-needed by default (e.g. https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/conf/distro/include/as-needed.inc?h=morty)
-ASNEEDED = ""
```

Comment:
No longer the case? This used to break the builds as Yocto adds `-Wl,--as-needed`. At least with Poky, did you build with Poky or RDK distro?

Reply:
I think this is merge trouble, I'll recover it.